File: xFit_rabbit/xFit_rabbit/Data/Recenzija.cs
using System;
using System.Collections.Generic;

namespace xFit_rabbit.Data;

public partial class Recenzija
{
    public int RecenzijaID { get; set; }

    public string? Sadrzaj { get; set; }

    public DateTime? Datum { get; set; }

    public int? ProizvodID { get; set; }

    public int? KorisnikId { get; set; }

    public int? KlijentId { get; set; }

    public virtual Klijent? Klijent { get; set; }

    public virtual Korisnik? Korisnik { get; set; }

    public virtual Proizvod? Proizvod { get; set; }
}

File: xFit/UI/xfit_mobilna/lib/models/recenzija.dart
import 'package:json_annotation/json_annotation.dart';

part 'recenzija.g.dart';

@JsonSerializable()
class Recenzija {
  int? recenzijaId;
  String? sadrzaj;
  DateTime? datum;
  int? proizvodId;
  int? klijentId;

  Recenzija(this.recenzijaId, this.sadrzaj, this.datum, this.proizvodId, this.klijentId);

  factory Recenzija.fromJson(Map<String, dynamic> json) => _$RecenzijaFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijaToJson(this);
}
